.github/workflows/pull_request_verification.yml

```diff
@@ -20,6 +20,19 @@ jobs:
       - name: Check correct commit message
         run: ./.github/scripts/commit/checkCommitMessage.sh
 
+  count-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR commits
+        run: echo "PR_FETCH_DEPTH=$(( ${{ github.event.pull_request.commits }} ))" >> "${GITHUB_ENV}"
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: ${{ env.PR_FETCH_DEPTH }}
+      - name: Count Commits
+        run: ./.github/scripts/commit/checkCountCommits.sh "${{ env.PR_FETCH_DEPTH }}"
+
```

Comment:
Try to call script directly here, seems that flow of passing to script and retrieving the result, only to execute simple assertion looks like an overengineering

Reply:
Yes, I agree with you it looks like overengineering. I changed it 😄